src/js/edu-benefits/1990-rjsf/config/form.js

```diff
@@ -24,20 +27,31 @@ const formConfig = {
   title: 'Apply for education benefits',
   subTitle: 'Form 22-1990',
   chapters: {
-    veteranInformation: {
-      title: 'Veteran Information',
+    applicantInformation: {
+      title: 'Applicant Information',
       pages: {
-        veteranInformation: {
-          title: 'Veteran information',
-          path: 'veteran-information',
+        applicantInformation: _.merge(applicantInformation(fullSchema1990, {
+          isVeteran: true,
+          fields: [
+            'veteranFullName',
+            'veteranSocialSecurityNumber',
+            'veteranDateOfBirth',
+            'gender'
+          ],
+        }), {
           uiSchema: {
-          },
-          schema: {
-            type: 'object',
-            properties: {
+            veteranDateOfBirth: {
+              'ui:validations': [
+                (errors, dob) => {
+                  // TODO: Use a constant instead of 'YYYY-MM-DD'
+                  if (moment(dob, 'YYYY-MM-DD').isAfter(moment().endOf('day').subtract(17, 'years'))) {
```

Comment:
This makes me a little nervous, since it implicitly relies on moment returning false when you call `isAfter` (or anything else, I think) on an invalid date (which is the case when `dob` is undefined).

Curious what you think. I worry that if we get used to doing this, we'll eventually hit a case where it does something unexpected.